### PR TITLE
Fixed 3 bugs

### PR DESCRIPTION
**Pull request by Anready**
Before:
Screenshot_2024-10-25-15-36-20-321_com coderGtm yantra pro
Screenshot_2024-10-25-15-36-35-488_com coderGtm yantra pro
Screenshot_2024-10-25-15-37-01-739_com coderGtm yantra pro
After:
Screenshot_2024-10-25-15-17-11-289_com coderGtm yantra pro debug
Screenshot_2024-10-25-15-16-49-685_com coderGtm yantra pro debug
Screenshot_2024-10-25-15-16-20-627_com coderGtm yantra pro debug

### CODE REVIEW

File: app/src/main/res/layout/custom_theme_dialog.xml [hunk 1]
```diff
@@ -29,6 +29,7 @@
                     android:layout_height="wrap_content"
                     android:layout_weight="1"
                     android:textSize="25dp"
+                    android:textColor="@color/white"
```
Comment (coderGtm) [inline app/src/main/res/layout/custom_theme_dialog.xml:32]: does this work when the device theme is light mode?

File: app/src/main/res/layout/custom_theme_dialog.xml [hunk 2]
```diff
@@ -52,7 +53,8 @@
                     android:layout_height="wrap_content"
                     android:layout_weight="1"
                     android:text="@string/input_color"
-                    android:textSize="25dp" />
+                    android:textSize="25dp"
```
Comment (coderGtm) [inline app/src/main/res/layout/custom_theme_dialog.xml:56]: same

File: app/src/main/res/layout/custom_theme_dialog.xml [hunk 3]
```diff
@@ -75,6 +77,7 @@
                     android:layout_height="wrap_content"
                     android:layout_weight="1"
                     android:textSize="25dp"
+                    android:textColor="@color/white"
```
Comment (coderGtm) [inline app/src/main/res/layout/custom_theme_dialog.xml:80]: same here also

File: app/src/main/res/layout/custom_theme_dialog.xml [hunk 4]
```diff
@@ -98,7 +101,8 @@
                     android:layout_height="wrap_content"
                     android:layout_weight="1"
                     android:text="@string/normal_text_and_arrow_color"
-                    android:textSize="25dp" />
+                    android:textSize="25dp"
```
Comment (coderGtm) [inline app/src/main/res/layout/custom_theme_dialog.xml:104]: ...

File: app/src/main/res/layout/custom_theme_dialog.xml [hunk 5]
```diff
@@ -98,7 +101,8 @@
...
+                    android:textColor="@color/white" />
```
Comment (coderGtm) [inline app/src/main/res/layout/custom_theme_dialog.xml:105]: ...

File: app/src/main/res/layout/custom_theme_dialog.xml (hunk 1 above)
Comment (Anready) [reply app/src/main/res/layout/custom_theme_dialog.xml:32]: hmm as much as i know on every theme background of alert dialog is black, and yes, my device on white theme

**Pull request merged**